src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,72 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+
+/**
+ * Represents the class that deals with loading tasks from file and saving tasks to file.
+ * 
+ * @author Sherman Ng Wei Sheng
+ */
+public class Storage {
+    private final String fileDir;
+    private final String filePath;
+
+    /**
+     * Constructor to initialise a new storage object.
+     * 
+     * @param filePath The path of the txt file that stores the data to be retrieved and to be updated.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        this.fileDir = this.filePath.substring(0,this.filePath.lastIndexOf("/") + 1);
+    }
+
+    /**
+     * Returns the string stored in the storage.
+     * 
+     * @return The string in the storage if it is existent or an empty string otherwise.
+     * @throws StorageLoadingException If error encountered during loading.
+     */
+    public String load() throws StorageLoadingException {
+        try {
+            File storageDir = new File(this.fileDir);
+            File storageFile = new File(this.filePath);
+            boolean isDirExistent = storageDir.exists();
```

Comment:
isDirExistent sounds a bit akward for me, maybe isDirPresent is a better alternative?